.storybook/preview.tsx

```diff
@@ -2,11 +2,16 @@ import '../app/style.css'
 import type { Preview } from '@storybook/react'
 import { initialize, mswLoader } from 'msw-storybook-addon'
 import * as MockDate from 'mockdate'
-import { initializeDB } from '#lib/db.mock'
-import { userEvent } from '@storybook/test'
+import { initializeDB } from '../lib/__mocks__/db'
+import { sb, userEvent } from 'storybook/test'
 initialize({ onUnhandledRequest: 'bypass', quiet: true })
 
-import { MINIMAL_VIEWPORTS } from '@storybook/addon-viewport'
+sb.mock('../app/actions', {spy: true});
+sb.mock('../lib/db', {spy: true});
+sb.mock('../lib/session', {spy: true});
+sb.mock('../lib/sanitize-html', {spy: true});
```

Comment:
Can we use:
```tsx
sb.mock(import('../app/actions'), {spy: true});
```

To make the import path more type safe?

Reply:
Shouldn't be a big issue to support both syntaxes